src/ng/plugins/groupbar/groupbar.html

```diff
@@ -14,7 +14,7 @@
 		<li>
 			<select ng-disabled="!$groupbar.add.canExecute()"
 					  ng-model="$groupbar.newGroup"
-					  ng-change="$groupbar.add.execute($groupbar.newGroup)">
+					  ng-change="$groupbar.add.execute($groupbar.newSort)">
```

Comment:
Revert name from newSort to newGroup (for src/ng/plugins/groupbar/groupbar.html file)